website/src/pages/partner-podcast.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { FunctionComponent } from 'react'
+import { Link, PageProps } from 'gatsby'
+
+import Layout from '../components/Layout'
+import { CaseStudyCard } from '../components/product/CaseStudyCard'
+import { BlogListItem } from '../components/BlogListItem'
+
+const resourceItems = [
+    {
+        title: 'Continuous developer onboarding: A guide to cultivating a culture of professional growth',
+        description:
+            'Download the guide to developer onboarding and learn how to shift to a culture of continuous onboarding in your engineering organization.',
+        type: 'Guide',
+        image: '/guides/dev-onboarding/hero.png',
+        href: '/guides/continuous-developer-onboarding',
+    },
+    {
+        title: 'Nutanix fixed Log4j quickly and confidently with Sourcegraph',
+        description:
+            'See how Nutanix was able to confidently identify every instance of Log4j across its sprwaling codebase and deliver patches to its customers that fully remediated the vulnerability within 4 days.',
+        type: 'Case study',
+        image: 'https://storage.googleapis.com/sourcegraph-assets/blog/code-insights-ga-blogs/code-insights-docs.png',
```

Comment:
To answer your question about images, I think it would be helpful for us to be provided with assets with consistent image dimensions. We could also crop/export however we need it from Figma and host this image ourselves in the about repo to match the spec.

Reply:
Totally. Okay I cropped and saved a thumbnail version of this image to use for scenarios like the Blog List Item 👍  @bretthayes

Reply:
I just checked out your changes. You'll probably need a 2x version of this and we'll have to size it down programatically. I know we don't have lazy loading or progressive image loading implemented yet, but this is something we should start to think about to provide higher quality.

Reply:
Good catch! Grabbed a higher quality, try now. Yea it'll be great to bring in lazy loading or progressive image loading in later if possible @bretthayes